src/libdredd/src/mutate_visitor.cc

```diff
@@ -214,6 +233,20 @@ bool MutateVisitor::TraverseStmt(clang::Stmt* stmt) {
     }
   }
 
+  // Some functions, e.g., __builtin_frame_address(), require their arguments to
+  // be constant integers. Attempting to apply a mutator function to such an
+  // argument would result in a compile-time error. Thus, we record these
+  // arguments' expressions so that they can be rewritten with the values to
+  // which the expressions would normally evaluate.
+  if (const auto* call_expr = llvm::dyn_cast<clang::CallExpr>(stmt)) {
+    const unsigned int frame_address_builtin_id = 455;
```

Comment:
How did you work out this was 455? Are you sure there isn't a named constant in some header file which yields this value? If so, it would be much better to use the named constant.

I did some digging and it seems that if you do:

`#include "clang/TargetBuiltins.h"`

then you should be able to use:

`clang::Builtin::BI__builtin_frame_address`

I presume that will evaluate to the required constant - can you check please?

Reply:
By running a program with __buitlin_frame_address and printing out the builtin id. Thank you for the suggestion, I didn't managed to find the named constant before.

changed to "#include "clang/Basic/TargetBuiltins.h", and rewrite to avoid the const variable declaration